src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 3 failures in `PowerShell-CI-linux`
Test-Connection.Ping.Allows us to Force IPv6
```powershell
Expected exactly '::1', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 116
116:             $result.Address | Should -BeExactly $targetAddressIPv6
```
Test-Connection.Ping.can convert IPv6 addresses to IPv4 with -IPv4 parameter
```powershell
Expected collection @('8.8.8.8', '8.8.4.4') to contain $null, but it was not found.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 125
125:             $result.Address.IPAddressToString | Should -BeIn @('8.8.8.8', '8.8.4.4')
```
Test-Connection.Ping.can convert IPv4 addresses to IPv6 with -IPv6 parameter
```powershell
Expected collection @('2001:4860:4860::8888', '2001:4860:4860::8844') to contain $null, but it was not found.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 134
134:             $result.Address.IPAddressToString | Should -BeIn @('2001:4860:4860::8888', '2001:4860:4860::8844')
```